clients/naming_client/naming_client.go

```diff
@@ -177,6 +181,7 @@ func (sc *NamingClient) SelectAllInstances(param vo.SelectAllInstancesParam) ([]
 	return service.Hosts, err
 }
 
+// SelectInstances 根据 DataId、Group、Health 获取所有实例信息
```

Comment:
chinese annotations need to be replaced with english annotations

Reply:
Translate all chinese in origin code ?

Reply:
Done! @sanxun0325